aws-junit/pom.xml

```diff
@@ -54,7 +54,7 @@
 
     <dependency>
       <groupId>org.elasticmq</groupId>
-      <artifactId>elasticmq-server_2.12</artifactId>
+      <artifactId>elasticmq-server_2.13</artifactId>
```

Comment:
while this is a test dep, I am just trying to avoid any build related stuff as we now match the support table for JDK 14 https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html cc @llinder